### ISSUE TITLE
My Twilio free trial ended

### ISSUE THREAD

**Issue by reginafcompton** [datamade/coordinated-entry-screening#31]
My Twilio free trial has ended! (The limit comes, not with time, but with usage.)

I have a call with the sales team on Monday, Sept. 17, 9:15 am. I'm going to see if they can extend my trial (for both my Gmail and DataMade email addresses). I'll also try to get more information about non-profit costs and relay that to Stephanie.

**Comment by reginafcompton** [comment]
I talked to a Twilio sales rep, who set-up a time for me to chat with someone in accounts. (Sigh.)

Fortunately, Twilio will be able to add $10 credit to my Gmail and DataMade accounts, which should extend the trial long enough to finish development.

I also learned about non-profit pricing for CSH. They need to complete an application on twilio.org, and then, Twilio can provide a $500 starter credit and 25% off in perpetuity. I'll relay that info to Stephanie.
